Error building the application

If you are using the _dotnet cli_ in _Windows_ to _build_ you get following error, related with the package _Microsoft.Web.LibraryManager.Build_, this is a [known issue](https://github.com/aspnet/LibraryManager/issues/514), i will update the package and make sure that still working as the same:

`User Path\.nuget\packages\microsoft.web.librarymanager.build\2.0.76\build\Microsoft.Web.LibraryManager.Build.targets(35,9): error MSB4018: The "Microsoft.Web.LibraryManager.Build.RestoreTask" task failed unexpectedly. [UserProjectPath\emplea_do\Web\Web.csproj]`

![image](https://user-images.githubusercontent.com/13991439/83214817-7ebde680-a133-11ea-90a1-94a79cdb4a8d.png)

**Steps to replicate the issue:**
1. Using the Terminal / cmd / Poweshell go to the path _emplea_do/web_
2. Run the command `dotnet run`

I have replicated in another windows environment with the same result.

--- a/Web/Web.csproj
+++ b/Web/Web.csproj
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="2.0.0-preview-009470001-12" />
     <PackageReference Include="Microsoft.FeatureManagement" Version="1.0.0-preview-009000001-1251" />
     <PackageReference Include="Scrutor" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="2.0.76" />
+    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="2.1.76" />
     <PackageReference Include="Flurl.Http" Version="2.4.2" />
   </ItemGroup>
   <ItemGroup>

